WalletWasabi.Gui/Controls/WalletExplorer/WalletViewModelBase.cs

```diff
@@ -23,8 +24,10 @@ public WalletViewModelBase(Wallet wallet)
 
 			_disposables = new CompositeDisposable();
 
-			Wallet = wallet;
 			Title = WalletName;
+			var isHardware = Wallet.KeyManager.IsHardwareWallet;
+			var isWatch = Wallet.KeyManager.IsWatchOnly;
+			TitleTip = isHardware ? "Hardware Wallet" : isWatch ? "Watch Only Wallet" : "Hot Wallet";
```

Comment:
Is this used somewhere? Are we instantiating pure WalletViewModelBase anywhere? If not then this can be just an empty string.

Reply:
It shows before you load a wallet so this makes sense